sass/components/_menu-dropdown.scss

```diff
@@ -34,9 +34,15 @@
                     &.cq-menu-dropdown-enter-done {
                         z-index: 6;
                     }
-                    .cq-categorical-display, .cq-dialog__body {
```

Comment:
@balakrishna-binary what's the issue of changing this line?

Reply:
`.cq-categorical-display` should not have height as 100% because it has a sibling `title div` which has height `40px`